assets/scss/main.scss

```diff
@@ -42,9 +57,20 @@ $component-subheader: (
 
 $component-input: (
 	theme: map-get($config, theme),
-	name: 'md-input-focused:not(.md-input-invalid) label',
+	name: 'md-input-focused:not(.md-input-invalid) > input',
+    border: map-get($config, theme),
 );
 
+$component-label: (
+	theme: map-get($config, theme),
+	name: 'md-input-focused:not(.md-input-invalid) > label',
+);
+
+
 @include component($component-toolbar);
+/*@include component($component-whiteframe); - Include if want cards to match the main background theme*/
+@include component($component-tileFooter);
```

Comment:
Name of variable `component-tileFooter` should be written in all lowercase letters with hyphens instead of underscores